packages/grafana-ui/src/components/Sparkline/Sparkline.tsx

```diff
@@ -118,9 +118,6 @@ export class Sparkline extends PureComponent<SparklineProps, State> {
       range: () => {
         const { sparkline } = this.props;
         if (sparkline.x) {
-          if (sparkline.timeRange && sparkline.x.type === FieldType.time) {
-            return [sparkline.timeRange.from.valueOf(), sparkline.timeRange.to.valueOf()];
```

Comment:
hymmm -- nervous about this change, especially for things like:
https://github.com/grafana/grafana/pull/37183

Is the issue that the timeRange is wrong?

Reply:
the queried/dashboard time range is larger than the time range of the data in the response.

if we want to support #37183, then we should keep the current behavior as-is and live with the gaps.